src/CustomItemInfo.cs

```diff
@@ -6,14 +6,24 @@ namespace custom_item_mod;
 [Serializable]
 public class CustomItemInfo
 {
+	// Required info fields
 	public string Name;
 	public string Description;
 	public int Amount;
 	public int Price;
 	
+	// Required asset references
 	public string AssetBundleName;
 	public string PrefabPath;
 	public string IconStandardPath;
 	public string IconDroppedPath;
-	public Vector3 PreviewBounds;
+
+	// Optional Asset References
+    public string ShelfPrefabPath;
```

Comment:
some inconsistent indentations in this file

Reply:
Fixed